examples/controlplane/controlplane-single/controlplane.yaml

```diff
@@ -0,0 +1,115 @@
+#####################################################
+# ${CLUSTER_NAME}-controlplane-0
+#####################################################
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-0
+  namespace: ${CLUSTER_NAME}
+  labels:
+    cluster.x-k8s.io/control-plane: "true"
+    cluster.x-k8s.io/cluster-name: "${CLUSTER_NAME}"
+spec:
+  version: ${KUBERNETES_VERSION}
+  bootstrap:
+    configRef:
+      apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+      kind: KubeadmConfig
+      name: ${CLUSTER_NAME}-controlplane-0
+      namespace: ${CLUSTER_NAME}
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: OpenStackMachine
+    name: ${CLUSTER_NAME}-controlplane-0
+    namespace: ${CLUSTER_NAME}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: OpenStackMachine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-0
+  namespace: ${CLUSTER_NAME}
+spec:
+  flavor: m1.medium
+  image: <Image Name>
+  sshKeyName: cluster-api-provider-openstack
+  availabilityZone: nova
+  networks:
+  - filter:
+      name: k8s-clusterapi-cluster-${CLUSTER_NAME}-${CLUSTER_NAME}
+    subnets:
+    - filter:
+        name: k8s-clusterapi-cluster-${CLUSTER_NAME}-${CLUSTER_NAME}
+  # multi-node control-plane:
+  # * Disable the floatingIP property
+  # single-node control-plane:
+  # * Enable the floatingIP property
```

Comment:
do we still want to keep those comments?

Reply:
Would delete it. Thanks